circle.yml

```diff
@@ -25,5 +25,5 @@ deployment:
     branch: master
     commands:
       - docker login -e "$DOCKER_REGISTRY_EMAIL" -u "$DOCKER_REGISTRY_USER" -p "$DOCKER_REGISTRY_PASSWORD"
-      - docker login -e '.' -u "$QUAY_USER" -p "$QUAY_PASSWORD" quay.io
-      - ./push-images
+      - if [ -n "$QUAY_PASSWORD" ]; then docker login -e '.' -u "$QUAY_USER" -p "$QUAY_PASSWORD" quay.io; fi
+      - ./push-images "$NOQUAY"
```

Comment:
Where is `$NOQUAY` set?  Did you mean to call with a blank string if not set? (Looks like this would lead to 'usage' message)